### PR TITLE
added support for .NET 6.0.9, .NET Core 3.1.29

### PR DESCRIPTION
**Pull request by waliMSFT**

### CODE REVIEW

File: build/__pythonVersions.sh [hunk 1]
```diff
@@ -9,4 +9,3 @@ PYTHON38_VERSION='3.8.12'
 PYTHON39_VERSION='3.9.7'
 PYTHON310_VERSION='3.10.4'
 PYTHON311_VERSION='3.11.0b1'
-
```
Comment (snehapar9) [inline build/__pythonVersions.sh:12]: Don't think we need these changes.

File: doc/supportedPlatformVersions.md [hunk 2]
```diff
@@ -757,6 +763,7 @@
 - 7.4.26
 - 7.4.28
 - 7.4.30
+- 8.0.22
 - 8.1.0
```
Comment (snehapar9) [inline doc/supportedPlatformVersions.md:767]: Do we need these platform versions to be added here?

**Review by snehapar9 — CHANGES_REQUESTED** [review]
Don't this we need these changes

File: build/__pythonVersions.sh (hunk 1 above)
Comment (waliMSFT) [reply build/__pythonVersions.sh:12]: after running the `generateconstants.sh` these changes are added. I will ignore this change.

File: doc/supportedPlatformVersions.md (hunk 2 above)
Comment (waliMSFT) [reply doc/supportedPlatformVersions.md:767]: auto generated..

File: build/__pythonVersions.sh (hunk 1 above)
Comment (waliMSFT) [reply build/__pythonVersions.sh:12]: auto generated.

File: build/__dotNetCoreRunTimeVersions.sh [hunk 3]
```diff
@@ -14,18 +14,18 @@ NET_CORE_APP_30='3.0.3'
...
 ASPNET_CORE_APP_50_SHA='d8e87804e9e86273c6512785bd5a6f0e834ff3f4bbebc11c4fcdf16ab4fdfabd0d981a756955267c1aa9bbeec596de3728ce9b2e6415d2d80daef0d999a5df6d'
-NET_CORE_APP_60='6.0.8'
-NET_CORE_APP_60_SHA='c776813bf87c25766b31a3a514d124d0526086ceea514a10f104d70ba435c91a6bd3c8bf10c6662b4df2b13ffcdf385518f3418e51d05cccec6a2cf2c26099de'
-ASPNET_CORE_APP_60='6.0.8'
-ASPNET_CORE_APP_60_SHA='b74676ca0d2f47a95533739fd36977bb1552890a81820ee51b29b3d6514398f0952362417bbb31fad4bdf031803cb3e8d2aaf065dfb154a78a1b471a536d4abd'
+NET_CORE_APP_60='6.0.9'
+NET_CORE_APP_60_SHA='a6df2cfef73047247bd36f51eaf696f616c6aa9b428e42f219bf91dcf05c03dff817a8ec826740002c8aa83df2fce8a7ace562ad2e2956789542f0b8ab8b1173'
+ASPNET_CORE_APP_60='6.0.9'
+ASPNET_CORE_APP_60_SHA='e808036155bc324335c309aaf948b2be1940a62eaf0135752989644698653c8f3a5ce310c3ee6742e3af73dbe175c6e529298eedf6eeb31cc38bfeab628f6d7b'
 NET_CORE_APP_70='7.0.0-preview.7.22375.6'
```
Comment (snehapar9) [inline build/__dotNetCoreRunTimeVersions.sh:29]: Think we need to update this as well to support .Net 7 RC1

**Review by snehapar9 — CHANGES_REQUESTED** [review]

File: doc/supportedPlatformVersions.md (hunk 2 above)
Comment (pauld-msft) [reply doc/supportedPlatformVersions.md]: Yeah looks like whenever php 8.0.22 was added for bullseye, the generateConstants script wasn't run. So now that it has run, we are seeing it resolve all the discrepancies from the versionsToBuild.txt files and this file

**Review by snehapar9 — DISMISSED** [review]

**Review by snehapar9 — DISMISSED** [review]

**Review by snehapar9 — APPROVED** [review]

**Pull request merged**